Assets/Scripts/Combat/CombatController.cs

```diff
@@ -40,7 +54,7 @@ public void BeginTurn()
         }
         isTurn = true;
         AssignCurrentTile();
```

Comment:
I would get rid of this line (it's already incorporated in FindSelectableBasicTiles).

Reply:
Good point, it's redundant.